### DISCUSSION TITLE
At every startup an icon appeare on the rigth side: Make not found. Why

### DISCUSSION THREAD

**Discussion by pinkfloyd11** [microsoft/vscode-discussions#447] [Extension Development QnA] [answered]
Hi 
At every startup an icon appeare on the rigth side: Make not found. 

If I click on "gear image" it correctly go on make extension.

**Comment by digitarald** [comment, marked as answer]
Sounds like you have an issue with your Make extension and view customizations.

I suggest reframing your question, so it can be answered in the context of extension development. Meanwhile I will close it as answered, but please feel free to unmark it.
